scripts.js

```diff
@@ -0,0 +1,6 @@
+var text = "Papugi koszą trawę, unosząc się nad ziemią 15 centymetrów, na czerwonych dywanach."
+var newText = "Zielone słonie"
+var animal = newText.toUpperCase()
+var textReplace = text.replace('Papugi', animal)
```

Comment:
Zrób tak aby w naszym `text` zmieniany był zawsze pierwszy wyraz, bez względu na to co tam będzie.

Możesz do tego wykorzystać metodę `split`